ligo/org/tool/ligo_proxy_init.py

```diff
@@ -222,6 +222,9 @@ def parse_args(parser):
     ):
         parser.error("one of username or -k/--kerberos is required")
 
+    if getattr(args, "username", None):
```

Comment:
```suggestion
    if hasattr(args, "username"):
```

Reply:
So I just changed it and blindly pushed, but it actually fails when username is empty (Kerberos) is used, so I reverted the change again:

```
AttributeError: 'Namespace' object has no attribute 'username'
```

Reply:
The key is to use `hasattr` instead of `getattr`, please reimplement the change as I proposed above (or just press the 'commit suggestion' button).